microsoft-365/admin/manage/set-password-expiration-policy.md

```diff
@@ -62,7 +62,7 @@ Here are some things to know about how this feature currently works as of Januar
     
 ## Prevent last password from being used again
 
-If you want to prevent your users from recycling old passwords, you can do so in Azure AD. See [Set the password expiration policy for your organization](https://docs.microsoft.com/office365/admin/manage/set-password-expiration-policy?view=o365-worldwide).
+If you want to prevent your users from recycling old passwords, you can do so in Azure AD. See [Enfore password history policy](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/enforce-password-history).
```

Comment:
```suggestion
If you want to prevent your users from recycling old passwords, you can do so in Azure AD. See [Enfore password history](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/enforce-password-history).
```